tests/integration/test_styles_reply_status_bar.py

```diff
@@ -5,20 +5,47 @@ def test_styles(mocker, main_window):
     wrapper = main_window.main_view.view_layout.itemAt(0).widget()
     conversation_scrollarea = wrapper.conversation_view.scroll
     reply_widget = conversation_scrollarea.widget().layout().itemAt(2).widget()
+    reply_widget.sender_is_current_user = False
 
     assert 5 == reply_widget.color_bar.minimumSize().height()
     assert 5 == reply_widget.color_bar.maximumSize().height()
-    assert "#0065db" == reply_widget.color_bar.palette().color(QPalette.Background).name()
+    assert "#9211ff" == reply_widget.color_bar.palette().color(QPalette.Background).name()
     # assert border: 0px;
 
-    reply_widget._set_reply_state("PENDING")
+    reply_widget.set_pending_styles()
 
     assert 5 == reply_widget.color_bar.minimumSize().height()
     assert 5 == reply_widget.color_bar.maximumSize().height()
     assert "#0065db" == reply_widget.color_bar.palette().color(QPalette.Background).name()
     # assert border: 0px;
 
-    reply_widget._set_reply_state("FAILED")
+    reply_widget.set_failed_styles()
+
+    assert 5 == reply_widget.color_bar.minimumSize().height()
+    assert 5 == reply_widget.color_bar.maximumSize().height()
+    assert "#ff3366" == reply_widget.color_bar.palette().color(QPalette.Background).name()
+    # assert border: 0px;
+
+
+def test_styles_for_replies_from_authenticated_user(mocker, main_window):
+    wrapper = main_window.main_view.view_layout.itemAt(0).widget()
+    conversation_scrollarea = wrapper.conversation_view.scroll
+    reply_widget = conversation_scrollarea.widget().layout().itemAt(2).widget()
+    reply_widget.sender_is_current_user = True
+
+    assert 5 == reply_widget.color_bar.minimumSize().height()
+    assert 5 == reply_widget.color_bar.maximumSize().height()
+    assert "#9211ff" == reply_widget.color_bar.palette().color(QPalette.Background).name()
+    # assert border: 0px;
```

Comment:
commented out throughout these tests, intentional?

Reply:
this was intentional at the time when we were aiming for 100% stylesheet test coverage but didn't have enough time to look into how to get the border attribute in Qt, etc., see https://github.com/freedomofpress/securedrop-client/issues/1097 <- when this issue is addressed we can remove these comments, but thinking about it now i think it makes sense to remove them since the comments are not very helpful without context

Reply:
Thanks for clarifying. I tend to use a `TODO` convention for this, which might be an alternative to removing, unless others prefer to not have those types of comments sticking around.

Reply:
that's a good convention. my hopes were that i would simply comment these lines one day when they decided to magically work on their own.